src/screens/CaseContactListScreen.tsx

```diff
@@ -1,12 +1,23 @@
-import { View, Text, FlatList } from 'react-native';
+import { View, Text, FlatList, SafeAreaView, Modal, Alert, Pressable, StyleSheet } from 'react-native';
 import tw from 'twrnc';
+import { useState, useEffect } from 'react'
+import React from 'react';
 
 import CaseContactListCard from '../components/CaseContactListCard';
+import { SafeAreaProvider } from 'react-native-safe-area-context';
 
-const CaseContactListScreen = ({ navigation }) => {
+const CaseContactListScreen = ({ navigation, route }) => {
+  const [modalVisible, setModalVisible] = useState(false);
   const data = [{ name: '🦋CINA-11-1002' }, { name: '🦋CINA-11-1003' }, { name: '🦋CINA-11-1004' }];
 
+  useEffect(() => {
+    if (route.params.staySignedIn) {
```

Comment:
take a look at "AccountScreen.tsx" for an example of how we can use the Context hook to fetch values from the state